libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -644,6 +646,26 @@ void RC_Channel::do_aux_function(const aux_func_t ch_option, const aux_switch_po
         do_aux_function_camera_trigger(ch_flag);
         break;
 
+    case AUX_FUNC::FENCE: {
+        // enable or disable the fence
+        AC_Fence *fence = AP::fence();
+        if (fence != nullptr) {
```

Comment:
This can be changed to 

```
if (fence == nullptr) {
    break;
}
```

and thus drop a level of indenting (this should be early-return in a new function)